### ISSUE TITLE
Handling alt-text fields of shared urls

### ISSUE THREAD

**Issue by lijocs** [DzmVasileusky/needShareButton#7]
![alt-text](https://user-images.githubusercontent.com/1256562/99155842-85540200-2670-11eb-8615-8171a35d4480.PNG)
Hi @DzmVasileusky,

Great job with the share button.

When I shared a url to Linkedin, and if the video image is invalid or not found, it shows **alt text** as thumbnail. An it displays **No alt text provided for this image.**. Is there any way, we could provide the alt text through your method. I tried to pass it as **description** field, but it didn't worked out.
